honeybadgermpc/batch_reconstruction.py

```diff
@@ -111,7 +126,8 @@ def with_at_most_non_none(data, k):
             yield x
 
 
-async def batch_reconstruct(elem_batches, p, t, n, myid, send, recv, debug=False):
+async def batch_reconstruct(elem_batches, p, t, n, myid, send, recv, debug=False,
```

Comment:
same comment as above

Reply:
There's actually a lot of code (at least in batch_reconstruction.py and polynomial.py) that follows this style. I believe it'll actually be harder to just split the arguments to a different line if a function has a lot of arguments.

Reply:
I guess it is okay. You can resolve comments related to these.

I checked PEP8 - this seems to be a permitted style.